networkx/algorithms/centrality/subgraph_alg.py

```diff
@@ -277,7 +277,7 @@ def communicability_betweenness_centrality(G):
         B[i, :] = 0
         B[:, i] = 0
         B -= np.diag(np.diag(B))
-        cbc[v] = float(B.sum())
+        cbc[v] = B.sum()
```

Comment:
In principle this has consequences for e.g. complex numbers; but since the current code will raise an exception and the changed code permits complex edge weights, I think that's fine.